### PR TITLE
OCPBUGS-41206: use monospace for tables values for better identification

### PR DESCRIPTION
**Pull request by jgbernalp**
This pr adds monospace as the default font family for tables so values can be identified better

Screenshot 2024-09-26 at 10 26 47

### CODE REVIEW

**Review by PeterYurkovich — CHANGES_REQUESTED** [review]

File: web/src/components/_monitoring.scss [hunk 1]
```diff
@@ -239,6 +239,10 @@ $screen-phone-landscape-min-width: 567px;
   width: 100%;
 }
 
+.monitoring-dashboards__table td {
+  font-family: monospace;
```
[the remark this answers is not in the document]
Comment (jgbernalp) [reply web/src/components/_monitoring.scss]: Thnx for the comment, It seems the RedHat Mono is slightly different from the regular monospace. Is fixed now.

**Comment by Tai-RedHat** [comment]
PR tested with cluster-bot, font changed
/label qe-approved

**Pull request merged**